utils/spacewalk-utils.spec

```diff
@@ -114,17 +138,57 @@ spacewalk-python3-pylint $RPM_BUILD_ROOT%{python3_sitelib}
 
 %files
 %defattr(-,root,root)
-%config %{_sysconfdir}/rhn/spacewalk-common-channels.ini
-%config(noreplace) %{_sysconfdir}/rhn/sw-ldap-user-sync.conf
-%attr(755,root,root) %{_bindir}/*
+%doc COPYING.GPLv2 COPYING.GPLv3
```

Comment:
AFAIK there is a new macro for license files (%license) which should be used.

Reply:
Right, this is just inherited and we need the new macro. Will fix it.

Reply:
Fixed.